ml-agents/mlagents/trainers/torch/networks.py

```diff
@@ -40,27 +40,29 @@ def __init__(
             else 0
         )
 
-        self.visual_encoders, self.vector_encoders = ModelUtils.create_encoders(
```

Comment:
It seems create_input_processors returns nn.Module so it does not make much sense to me to use the names `self.visual_inputs` I think `self.visual_processor` is better if we are going for processor instead of encoder.